packages/terra-worklist-data-grid/src/proptypes/rowShape.js

```diff
@@ -0,0 +1,15 @@
+import PropTypes from 'prop-types';
+import cellShape from './cellShape';
+
+const rowShape = PropTypes.shape({
+  /**
+   * String that specifies the row height. Any valid CSS height value is accepted.
```

Comment:
Should each row have an id as exists in the current API?

Reply:
At this point, we don’t have the need for it, but we will introduce it to accommodate future features such as row selection etc.